### ISSUE TITLE
zsh’s antibody causes illegal instruction

### ISSUE THREAD

**Issue by NightMachinary** [ish-app/ish#709]
```
\h:\w\$ curl -sfL git.io/antibody | sh -s - -b /usr/local/bin
getantibody/antibody info checking GitHub for latest tag
getantibody/antibody info found version: 6.0.1 for v6.0.1/Linux/i386
getantibody/antibody info installed /usr/local/bin/antibody
\h:\w\$ antibody bundle zsh-users/zsh-autosuggestions
zsh: illegal hardware instruction  antibody bundle zsh-users/zsh-autosuggestions
\h:\w\$ dmesg 
[    0.000000] 15 stub syscall 375
               41 stub syscall 375
               53 stub syscall 375
               56 stub syscall 375
               74 illegal instruction at 0x809bac5: f2 0f 59 c1 f2 0f 58 44
```

**Comment by tbodt** [comment]
Blocked on #698

**Comment by tbodt** [comment]
The instructions Golang uses now seem to be implemented, so closing this now. If it still doesn't work in the next update, please open a new issue.
